### PR TITLE
Add config file

### PR DESCRIPTION
**Pull request by gemmai95**
Add all config for service to be with app this should make easier to maintain and change.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[Continue to create config.ini files for SDX services](https://trello.com/c/YaVmIhT3/481-continue-to-create-configini-files-for-sdx-services)

**Comment by gemmai95** [comment]
[Continue to create config.ini files for SDX services](https://trello.com/c/YaVmIhT3/481-continue-to-create-configini-files-for-sdx-services)

File: config.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+LOGGING_LEVEL: DEBUG
+
+[Service URLs]
+SDX_SEQUENCE_URL = http://sdx-sequence:5000
+
+[FTP setup]
+FTP_HOST: \\\\NP3RVWAPXX370\\SDX_preprod\\
+SDX_FTP_IMAGES_PATH: "EDC_QImages"
+SDX_FTP_DATA_PATH: "EDC_QData"
+SDX_FTP_RECEIPT_PATH: "EDC_QReceipts"
```
Comment (insacuri) [inline config.ini]: Add newline character

File: config.ini [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[Default]
+PORT: 8084
+HOST: 127.0.0.1
```
Comment (insacuri) [inline config.ini]: Can be removed (sdx_transform_cora_url will use different variables to be figured out)

File: config.ini [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+LOGGING_LEVEL: DEBUG
+
+[Service URLs]
+SDX_SEQUENCE_URL = http://sdx-sequence:5000
+
+[FTP setup]
+FTP_HOST: \\\\NP3RVWAPXX370\\SDX_preprod\\
+SDX_FTP_IMAGES_PATH: EDC_QImages
+SDX_FTP_DATA_PATH: EDC_QData
+SDX_FTP_RECEIPT_PATH: EDC_QReceipts
```
Comment (insacuri) [inline config.ini]: Newline char still isn't there >_<

**Review by insacuri — APPROVED** [review]

**Comment by gemmai95** [comment]
[sdx-transform-cora - implement py.test](https://trello.com/c/zGYEn70K/501-sdx-transform-cora-implement-pytest)

**Pull request merged**